src/earn/EarnActivePools.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { EarnEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { formatValueToDisplay } from 'src/components/TokenDisplay'
+import { useDollarsToLocalAmount } from 'src/localCurrency/hooks'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { useSelector } from 'src/redux/hooks'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import Logger from 'src/utils/Logger'
+import networkConfig from 'src/web3/networkConfig'
+
+const TAG = 'earn/EarnActivePools'
+
+export default function EarnActivePools() {
+  const { t } = useTranslation()
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  // TODO(ACT-1268): use info from getEarnPositions
+  const poolToken = useTokenInfo(networkConfig.aaveArbUsdcTokenId)
+  const poolTokenInLocalCurrency = useDollarsToLocalAmount(poolToken?.balance ?? null)
+  if (!poolToken) {
+    // should never happen
+    Logger.error(TAG, `No pool token found ${networkConfig.aaveArbUsdcTokenId}`)
+    return null
+  }
+  const poolsSupplied = 1
+  const totalSupplied = `${localCurrencySymbol}${poolTokenInLocalCurrency ? formatValueToDisplay(poolTokenInLocalCurrency) : '--'}`
+
+  return (
+    <View style={styles.card} testID="EarnActivePools">
+      <View style={styles.container}>
+        <Text style={styles.title}>{t('earnFlow.activePools.title')}</Text>
+        <View>
+          <View style={styles.row} testID="EarnActivePools/PoolsSupplied">
+            <Text style={styles.labelText}>{t('earnFlow.activePools.poolsSupplied')}</Text>
+            <Text style={styles.valueText}>{poolsSupplied}</Text>
+          </View>
+          <View style={styles.separator} />
+          <View style={styles.row} testID="EarnActivePools/TotalSupplied">
+            <Text style={styles.labelText}>{t('earnFlow.activePools.totalSupplied')}</Text>
+            <Text style={styles.valueText}>{totalSupplied}</Text>
+          </View>
+        </View>
+        <View style={styles.buttonContainer}>
+          <Button
+            onPress={() => {
+              ValoraAnalytics.track(EarnEvents.earn_active_pools_cta_press, {
+                action: 'exploreOpenPools',
```

Comment:
should these actions be pulled out into an enum?

Reply:
I think string literals are fine, we talked about enums vs string literals in one of the eng syncs and I don't think we landed on using only one over the other. Enums seem like an overkill for this case